README.md

```diff
@@ -1,11 +1,23 @@
 # InfluxDB [![CircleCI](https://circleci.com/gh/influxdata/influxdb.svg?style=svg)](https://circleci.com/gh/influxdata/influxdb)
 
-This is the for InfluxDB 2.0 OSS, which includes parts that used to be known as Chronograf and Kapacitor.
+InfluxDB is an open source time series platform. This includes APIs for storing and querying data, processing it in the background for ETL or monitoring and alerting purposes, user dashboards, and visualizing and exploring the data and more. The master branch on this repo now represents InfluxDB 2.0, which includes functionality for Kapacitor (background processing) and Chronograf (the UI). If you are looking for the 1.x line of releases, there are branches for each of those. InfluxDB 1.8 will be the next (and likely last) release in the 1.x line and the [working branch is here](https://github.com/influxdata/influxdb/tree/1.8).
 
-If you are looking for the [InfluxDB 1.x Go Client, we've created a new repo](https://github.com/influxdata/influxdb1-client) for that.
+If you are looking for the [InfluxDB 1.x Go Client, we've created a new repo](https://github.com/influxdata/influxdb1-client) for that. There will be a Go client for the 2.0 API coming very soon.
+
+## State Of the Project
+
+InfluxDB 1.7.3 is the current stable release and recommended for production use. InfluxDB 2.0 (what's in the master branch) is currently in the alpha stage. This means that it is not recommended for production usage. There may be breaking API changes, breaking changes in the [Flux language](https://github.com/influxdata/flux), changes in the underlying storage format that will require you to wipe out all your old data, and significant changes to the UI. We will be cutting weekly versioned releases every week starting in the first week of February. There will also be nightly builds.
```

Comment:
I would prefer not to put this complete version number here as it will be out of date month-to-month and I don't want to update this README with each release.

Reply:
"weekly versioned releases every week" redundant

Reply:
I would also suggest adding something like:

The alpha is intended for feature exploration and gathering feedback on the available feature set.  It should not be used for performance testing, benchmarks, or other stress tests.  Additional features will arrive during the weekly alpha updates.